verification/curator-service/ui/src/components/Users.test.tsx

```diff
@@ -136,14 +135,9 @@ test('updates roles on selection', async () => {
     expect(mockedAxios.put).toHaveBeenCalledWith('/api/users/abc123', {
         roles: ['admin', 'reader', 'curator'],
     });
-    expect(await findByText('Alice Smith')).toBeInTheDocument();
```

Comment:
Why remove these checks?

Reply:
contents are checked in the cypress tests, didn't like the redundancy and the fact that we have to mock again and again the axios responses here, we're just checking that the document contains whatever we put in our mocks in unit tests which feels useless to me.